tests/internal/e2e/test/specs/vcast.test.ts

```diff
@@ -491,7 +491,7 @@ describe("vTypeCheck VS Code Extension", () => {
     ).toBe(true);
     expect(
       outputViewText.includes(
-        "test explorer  [info]  Test summary for: cpp/unitTests/DATABASE-MANAGER|manager.Manager::PlaceOrder.myFirstTest",
+        "test explorer  [info]  Test summary for: vcast:cpp/unitTests/DATABASE-MANAGER|manager.Manager::PlaceOrder.myFirstTest",
```

Comment:
@johnpaliotta didn't we say that the `vcast:` prefix was something "internal and hidden" and wasn't supposed to be user-visible?

Reply:
It is not "seen" except in this logging context, which I think is ok